Chart.yaml

```diff
@@ -49,8 +49,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.22.0
+version: 0.22.1
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 1.20.2
```

Comment:
why they are different before?

Reply:
I don't know.

Reply:
`version` refers to charts and templates changes, so I don't think there's any requirements to have them similar.